client/state/data-layer/wpcom/sites/automated-transfer/status/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { delay } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { AUTOMATED_TRANSFER_STATUS_REQUEST } from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import {
+	getAutomatedTransferStatus,
+	setAutomatedTransferStatus,
+} from 'state/automated-transfer/actions';
+
+export const requestStatus = ( { dispatch }, action ) => {
+	const { siteId } = action;
+
+	dispatch( http( {
+		method: 'GET',
+		path: `/sites/${ siteId }/automated-transfers/status`,
+		apiVersion: '1',
+	}, action ) );
+};
+
+export const receiveStatus = ( { dispatch }, { siteId }, next, { status, uploaded_plugin_slug } ) => {
+	dispatch( setAutomatedTransferStatus( siteId, status, uploaded_plugin_slug ) );
+	if ( status !== 'complete' ) {
+		delay( dispatch, 3000, getAutomatedTransferStatus( siteId ) );
+	}
+};
+
+export default {
+	[ AUTOMATED_TRANSFER_STATUS_REQUEST ]: [ dispatchRequest(
+		requestStatus,
+		receiveStatus
+	) ]
```

Comment:
It looks like all 3 parameters are required for `dispatchRequest`. The error handler is missing.

Try `dispatch( { type: 'AUTOMATED_TRANSFER_STATUS_REQUEST', siteId: 'somebadid' } )`, which leads to a `typeError`.

Does some error handling make sense here? It could be `noop` for now, but perhaps showing a notice in the future makes sense.

Reply:
Yeah, I'm not sure what kind of error handling we want here. A notice on every failed poll seems like it could get messy?

I'll put in a `noop` for now. Other suggestions welcome.

Reply:
`noop` should be just fine for now; I doubt we'd want to show a message to the user in that case for now.